src/api/verify.ts

```diff
@@ -18,100 +19,105 @@ let expectedStateCode: string | null = null;
 
 class VerifyAPI {
   static async trackVerified(params: RadarTrackVerifiedParams, encrypted: Boolean = false) {
-    const options = Config.get();
-
-    const { skipVerifyApp } = params;
-
-    // user indentification fields
-    const userId = params.userId || Storage.getItem(Storage.USER_ID);
-    const deviceId = Device.getDeviceId();
-    const installId = Device.getInstallId();
-    const sessionId = Session.getSessionId();
-    const description = params.description || Storage.getItem(Storage.DESCRIPTION);
-
-    // save userId
-    if (!userId) {
-      Logger.warn('userId not provided for trackVerified.');
-    } else {
-      Storage.setItem(Storage.USER_ID, userId);
-    }
+    try {
```

Comment:
Wraps in try/catch and propagates errors to `onError()` callback before throwing again

Needs to be done inside this function (instead of in `api.ts`) so that we can propagate errors both for `trackVerified()` and `startTrackingVerified()`